### PR TITLE
Feat#1262: Index user group data to search microservice

### PR DESCRIPTION
**Pull request by ghanlohar**
# Closes/Fixes/Resolves

### 
ONEPLAT-1262
# Explain the feature/fix

## Does this PR introduce a breaking change

- Indexes user group data to search microservice
- Minor correction in the helper code for feedback service

Yes

## Screenshots

<details>
<summary>View Screenshots</summary>

Screenshot 2021-07-14 at 11 36 54 AM

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Review by riginoommen — APPROVED** [review]

File: packages/user-group-service/src/groups/resolver.ts [hunk 1]
```diff
@@ -39,17 +39,35 @@ export const GroupResolver = {
...
+        });
     },
     deleteGroup ( root: any, { id }: GraphQLArgs, ctx: any ) {
       return Groups
         .findByIdAndRemove( id )
-        .exec();
+        .exec()
+        .then( ( res: any ) => {
+          const input = {
+            dataSource: "oneportal",
```
Comment (deshmukhmayur) [inline packages/user-group-service/src/groups/resolver.ts:65]: We should probably use an environment variable for the data source.

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]

File: packages/user-group-service/src/groups/resolver.ts (hunk 1 above)
Comment (ghanlohar) [reply packages/user-group-service/src/groups/resolver.ts:65]: @riginoommen what do you think about it?

Comment (riginoommen) [reply packages/user-group-service/src/groups/resolver.ts:65]: I think we can use a global environment variable. I will be good also. I agree with it.

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: packages/feedback-service/.env.example [hunk 2]
```diff
@@ -33,3 +33,5 @@ EMAIL_PORT=
 
 ## URLs
 FEEDBACK_CLIENT=
+
+DATA_SOURCE=
```
Comment (riginoommen) [inline packages/feedback-service/.env.example:37]:
```suggestion
SEARCH_DATA_SOURCE=
```

File: packages/feedback-service/src/helpers.ts [hunk 3]
```diff
@@ -317,9 +317,9 @@ class FeedbackHelper {
     public formatSearchInput(data: any) {
         return {
             'input': {
-                'dataSource': 'oneportal',
+                'dataSource': process.env.DATA_SOURCE,
```
Comment (riginoommen) [inline packages/feedback-service/src/helpers.ts:320]:
```suggestion
                'dataSource': process.env.SEARCH_DATA_SOURCE,
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

**Review by deshmukhmayur — APPROVED** [review]

**Review by hybridx — APPROVED** [review]

**Review by riginoommen — APPROVED** [review]

**Pull request merged**